webpack.config.js

```diff
@@ -11,13 +11,15 @@ module.exports = function (env = {}) {
       'zaf_sdk': [
         'native-promise-only',
         './lib/index.js'
-      ],
```

Comment:
I moved this to the `if production` condition, because for tests this was taking forever.